pkg/identity/numericidentity.go

```diff
@@ -55,8 +59,86 @@ const (
 	// ReservedIdentityInit is the identity given to endpoints that have not
 	// received any labels yet.
 	ReservedIdentityInit
+
+	// --------------------------------------------------------------
+	// Special identities for well known cluster components
+
+	// ReservedCiliumKVStore is the reserved identity used for the kvstore
+	// managed by Cilium (etcd-operator).
+	ReservedCiliumKVStore NumericIdentity = 100
+
+	// ReservedKubeDNS is the reserved identity used for kube-dns.
+	ReservedKubeDNS NumericIdentity = 101
+
+	// ReservedEKSKubeDNS is the reserved identity used for kube-dns on EKS
+	ReservedEKSKubeDNS NumericIdentity = 102
 )
 
+type wellKnownIdentities struct {
```

Comment:
Any specific reason reason for not going with

`type wellKnownIdentities map[NumericIdentity]wellKnownIdentity`
?

Reply:
Yes, a struct allows being extended later on without changing all existing code.